Update ReleaseProcess.md (#297)
Babylonpartners/ios-playbook: Cookbook/Technical-Documents/ReleaseProcess.md

### Phase 2: Initiation
*It starts at the end of the sprint (typically when the new sprint starts on Monday)*

> 

Reviewer: if we do not do automatic cut-offs for other targets then this step should not be removed

Author: 🤦‍♂

Author: please use github suggestions @viorel15

1. Release branch is cut automatically on last Friday of the sprint during nightly builds. It will create and push release branch, create a release Slack channel, submit a new appcenter build and run UI tests.

>    * if for any reason automatic cut didn't work it can be triggered manually with `/stevenson release_cutoff target:Babylon` fro #ios_build channel

Reviewer 2: ```suggestion
   * if, for any reason, the automatic cut didn't work it can be triggered manually with `/stevenson release_cutoff target:Babylon` from `#ios-build` channel
```

>    * if, for any reason, the automatic cut didn't work it can be triggered manually with `/stevenson release_cutoff target:Babylon` from `#ios-build` channel

Author: ```suggestion
   * if, for any reason, the automatic cut didn't work it can be triggered manually with `/stevenson release_cutoff target:Babylon` from `#ios-build` channel
   * If you're releasing another app (e.g. Telus, Bupa, NHS111), since they typically go thru the release process _only_ once the main Babylon app has been signed off by QA, you should create the new branch from the corresponding Babylon release branch that was recently already QA'd and signed off (e.g. `release/babylon/4.1.0`) instead of `develop`, and name your new branch using the same `release/{appname}/{version}` convention (e.g. `release/telus/4.1.0`)
```
